Cargo.toml

```
@@ -604,6 +604,8 @@ x509-parser = "0.14.0"
 # See "zeroize versioning issues" below if you are updating this version.
 zeroize = { version = "1.7", default-features = false }
 zstd = "0.13.3"
+rocksdb = { version = "0.23.0", default-features = false }
+serde_yaml_08 = "0.8.26"
```

I think it should be:

```suggestion
serde_yaml_08 = { package = "serde_yaml", version = "0.8.26" }
```